src/main/kotlin/com/openlattice/chronicle/services/studies/StudyService.kt

```diff
@@ -136,4 +139,11 @@ class StudyService(
 
             }
     }
+
+    override fun deleteStudies(connection: Connection, studyIds: Collection<UUID>): Int {
+        val ps = connection.prepareStatement(DELETE_STUDIES_SQL)
+        val pgStudyIds = PostgresArrays.createUuidArray(ps.connection, studyIds)
+        ps.setArray(1, pgStudyIds)
+        return ps.executeUpdate()
+    }
```

Comment:
This might be ok but it's very different from the exiting logic. Currently, deleting a study also means deleting all participants as well as all data, i.e. neighbors.

I assume we want to preserve that behavior.

Reply:
I see. I didn't know about that or how participants might work/change in v3.

Reply:
Deleting a study should delete all of the study data. There also needs to be a way to pause study data collection (this might require android changes-- I believe there is an isRunning API, but it is server wide and doesn't take any parameters). See second requirement here: https://openlattice.atlassian.net/wiki/spaces/ENGR/pages/2120351745/Chronicle

Reply:
So you'll need to (1) delete all usage events in redshift (there's a study id column), delete the organization_studies entries for that study if any, delete the actual study entry.  Every thing in platform storage can be done as a single transaction. Redshift will need separate transaction.

Reply:
This will need to be an async operation. Historically, deleting a single participant with sufficiently large data takes forever. The api request needs to be quick even if the actual data takes hours (or days) to actually be deleted.